app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryAttachments.js

```diff
@@ -58,40 +58,45 @@ class ExpandedStoryAttachments extends React.Component {
   render() {
     const { story, onDelete } = this.props;
 
+    if(!canEdit(story) && !story.documents.length) return null
+
     return (
       <ExpandedStorySection
         title={I18n.t('story.attachments')}
         identifier="attachments"
       >
-        <Dropzone
-          onDrop={this.onFileDrop}
-          accept={acceptedMimeTypes()}
-          disabled={this.state.loading}
-          multiple={false}
-        >
-          {({ getRootProps, getInputProps, isDragActive, isDragReject }) => {
-            const className = this.dropzoneClassName({ isDragActive, isDragReject })
-
-            return (
-              <div
-                {...getRootProps()}
-                className={`btn btn-success attachments-dropzone ${className}`}
-              >
-                <input {...getInputProps()} />
-                <i className="mi md-20">cloud_upload</i>
-                <div>
-                  {!isDragReject ?
-                    I18n.t('upload_new_file') :
-                    I18n.t('reject_new_file')}
-                </div>
-              </div>
-            )
-          }}
-        </Dropzone>
+        {
+          canEdit(story) && 
```

Comment:
We don't need this `canEdit(story)` here because we already return `null` if it can't edit in line 61 😄